netlify.toml

```diff
@@ -2,6 +2,12 @@
   publish = "dist"
   command = "npm run build-docs"
 
+# 404 status
```

Comment:
**issue (blocking)**: this rule is catching everything before it has a chance to be redirected by one of the more specific rules below. Visiting https://protocol.mozilla.org/fundamentals/principles.html redirects to https://protocol.mozilla.org/docs/fundamentals/principles but https://demo2--mozilla-protocol.netlify.app/fundamentals/principles.html shows 404 instead of redirect

<img width="1077" alt="missing-redirect" src="https://user-images.githubusercontent.com/19650432/223769561-d14d3581-8226-457b-b60b-bb1004935131.png">

**suggestion**: move wildcard 404 rule to the bottom so it only catches urls that don't match anything more specific

Reply:
Thanks, eagle eye! 🦅 👁️